src/adapter/src/notice.rs

```diff
@@ -236,6 +240,7 @@ impl AdapterNotice {
             }
             AdapterNotice::UserRequested { .. } => SqlState::WARNING,
             AdapterNotice::ClusterReplicaStatusChanged { .. } => SqlState::WARNING,
+            AdapterNotice::CascadeDroppedObject { .. } => SqlState::WARNING, // unsure
```

Comment:
Just want to verify that a dropped notice is considered a warning.

Reply:
Actually I had a slight change of heart, let's change this error code to `SqlState::SUCCESSFUL_COMPLETION` and we can fix the existing ones in a follow up.

Reply:
FYI: https://github.com/MaterializeInc/materialize/issues/23313

Reply:
> For now, let's keep this as `SqlState::SUCCESSFUL_COMPLETION` to keep this consistent with the rest of the code and I'll open up a follow-up issue to address this discrepancy.

Thanks Joe, will do. Happy to work on that issue afterwards if needed

Reply:
> > For now, let's keep this as `SqlState::SUCCESSFUL_COMPLETION` to keep this consistent with the rest of the code and I'll open up a follow-up issue to address this discrepancy.
> 
> Thanks Joe, will do. Happy to work on that issue afterwards if needed

If you're interested, then go for it.